SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -62,29 +67,30 @@ class USpatialReceiver : public UObject, public SpatialOSDispatcherInterface
 	GENERATED_BODY()
 
 public:
-	void Init(USpatialNetDriver* NetDriver, FTimerManager* InTimerManager, SpatialGDK::SpatialRPCService* InRPCService);
+	void Init(USpatialNetDriver* NetDriver, FTimerManager* InTimerManager, SpatialGDK::SpatialRPCService* InRPCService,
+			  SpatialGDK::SpatialEventTracer* InEventTracer);
 
 	// Dispatcher Calls
 	virtual void OnCriticalSection(bool InCriticalSection) override;
-	virtual void OnAddEntity(const Worker_AddEntityOp& Op) override;
+	virtual void OnAddEntity(const Worker_Op& Op) override;
```

Comment:
The Ops have SpanIds on them that allow us to link up events. Not sure how this will work with spatial view. Perhaps we have to cache these SpanIds similarly to what we do with component updates. Perhaps they can be propagated through spatial view.

Reply:
Even in the current form of spatial view anything using ops that isn't a command for before the view delta is created won't actually do anything. But it will conflict with new code and have to be resolved.